glean-core/python/tests/test_network.py

```diff
@@ -81,11 +81,12 @@ def test_500_error_submit(safe_httpserver, monkeypatch):
     ProcessDispatcher._wait_for_last_process()
 
     # This kind of recoverable error will be tried 10 times
-    assert 10 == len(safe_httpserver.requests)
+    # The number of retries is defined on glean-core
+    assert 3 == len(safe_httpserver.requests)
```

Comment:
Ah, was it retried 10 times before in Python?

Reply:
Uhum, actually this was a bug in the `MAX_RETRIES` python implementation. And subsequently a bug in the tests, actually it wouldn't only be retried 10 times, there was no limitation.

I was not taking into account the the recoverable failures related to 5xx status codes. 

+1 for centralizing code and removing duplication I guess.

Reply:
Remider for myself to update the comment in line 83 here.

Reply:
I see. If this was a bug in Python, would you kindly add a related BUGFIX entry in the changelog?